src/fhirtypes/CodeSystem.ts

```diff
@@ -10,6 +11,7 @@ import { applyMixins } from '../utils';
  * @see {@link https://www.hl7.org/fhir/codesystem.html}
  */
 export class CodeSystem {
+  readonly resourceType = 'CodeSystem';
```

Comment:
You'll see I added a `resourceType` property to several of our classes.  This makes it easier to identify the type at run-time, and is used when writing out the files.